scripts/build/parse_features_toc.py

```diff
@@ -51,15 +51,15 @@ def createVersionHref(parent, tocHref, tocString):
 
 # Only process the features of the highest version for the docs draft site
 if(os.getenv("DRAFT_SITE") or os.getenv("DOCS_DRAFT_SITE")):
-    max = ['0', '0', '0', '0']
-    for version in versions:
-        nums = version.split('.')
-        if(int(nums[0]) > int(max[0])):
-            max = nums
-        elif(int(nums[0]) == int(max[0]) and int(nums[3]) > int(max[3])):
-            max = nums
-    max = '.'.join(max)
-
+    # max = ['0', '0', '0', '0']
```

Comment:
Here the commented code Only process the features of the highest version for the docs draft site.If we were using older version of antora here the max value will be come as 22.0.0.3 because that is the highest version now. But with antora update when we add this line in docs-playbook.yml
urls:
  latest_version_segment: latest
the highest version will be automatically convert as latest (eg: here 22.0.0.3 will be coming as latest)
 So since this commented code is only to find highest version that now antora is doing for us and will come as latest